scRNA-seq/04-dimension_reduction_scRNA.Rmd

```diff
@@ -131,29 +134,31 @@ hodgkins_sce <- hodgkins_sce[, hodgkins_sce$total > 0]
 dim(hodgkins_sce)
 ```
 
-### Filtering with `emptyDrops()`
+### Filtering with `emptyDropsCellRanger()`
 
 The `DropletUtils` package that we used to read in the 10X data has a number of other useful features.
-One is the `emptyDrops()` function, which uses the overall gene expression patterns in the sample to identify droplets that are likely to not contain an intact cell, but may simply have contained loose ambient RNA released during cell separation.
+One is the `emptyDropsCellRanger()` function, which uses the overall gene expression patterns in the sample to identify droplets that are likely to not contain an intact cell, but may simply have contained loose ambient RNA released during cell separation.
+This method was originally developed by [Lun *et al.* (2019)](https://doi.org/10.1186/s13059-019-1662-y) and implemented as the function `emptyDrops()`, but has since been adapted as the main filtering method used by Cell Ranger. 
+The `emptyDropsCellRanger()` function emulates the variant of this function that used by Cell Ranger, making the results more comparable between the two methods.
 
-This method, described in [Lun *et al.* (2019)](https://doi.org/10.1186/s13059-019-1662-y), uses the droplets with very low UMI counts to estimate the "ambient" expression pattern of RNA content outside of cells.
+This method, described in , uses the droplets with very low UMI counts to estimate the "ambient" expression pattern of RNA content outside of cells.
```

Comment:
accidental deletion here?

Reply:
Incomplete deletion, it looks like. Updated with more specificity.